.drone/README.md

```diff
@@ -18,3 +22,21 @@ Alternatively, run the following `make` command from the project root directory:
 ```
 make drone
 ```
+
+## Drone CI environment
+
+The test coverage step requires the `jq` tool to run. The `jq` tool is provided by the `Dockerfile` in this directory. If you need to update the image, make your changes to the `Dockerfile` and run the following to push the new image to GCR:
+
+```
+docker build --platform linux/amd64 -t i2c/drone-env:latest .
+
+docker tag i2c/drone-env:latest us.gcr.io/kubernetes-dev/influx2cortex/drone-env:latest
+
+docker push us.gcr.io/kubernetes-dev/influx2cortex/drone-env:latest
+```
+
+Alternatively, run the following `make` command forem the project root directory:
+
+```
+make drone-env
```

Comment:
same as above, if the make command takes care of the details, then no need to document the details :)